debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/Filters.java

```diff
@@ -32,8 +32,8 @@
      * @param config the configuration; may not be null
      */
     public Filters(Configuration config) {
-        String dbIncludeList = config.getFallbackStringProperty(MongoDbConnectorConfig.DATABASE_INCLUDE_LIST, MongoDbConnectorConfig.DATABASE_WHITELIST);
-        String dbExcludeList = config.getFallbackStringProperty(MongoDbConnectorConfig.DATABASE_EXCLUDE_LIST, MongoDbConnectorConfig.DATABASE_BLACKLIST);
+        String dbIncludeList = config.getFallbackStringProperty(config, MongoDbConnectorConfig.DATABASE_INCLUDE_LIST, MongoDbConnectorConfig.DATABASE_WHITELIST);
```

Comment:
@gunnarmorling Should we really call an instance method and pass the instance as parameter to that method? 
That's why I added the instance method with a cleaner parameter list and the static one which actually handled the code.

Reply:
Hum, no. I was expecting `config.getFallbackStringProperty( MongoDbConnectorConfig.DATABASE_INCLUDE_LIST, MongoDbConnectorConfig.DATABASE_WHITELIST);`. Passing the config instance again to an instance method doesn't make sense.